src/System.Collections/tests/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
       netstandard;
+      netcoreapp;
```

Comment:
nit: I would keep netcoreapp above netstandard like it was before (same with the configurations in the project file).